pages/v/[id].tsx

```diff
@@ -100,6 +100,10 @@ const Playback: React.FC<Props> = ({ playbackId, shareUrl, poster }) => {
         .error-message {
           color: #ccc;
         }
+        .loading {
+          font-size: 8vw;
+          max-width: 100%;
+        }
```

Comment:
I can see that `FullpageLoader` already has these styles `font-size: 8vw; max-width: 100%;` applied to it within the component, so I'm going to remove the `className` and these styles here (that's what Typescript was complaining about)

Reply:
Ahh I see, yeah something weird seemed to be happening with the "Loading player" text and I thought this fixed it but maybe it just complicated it more